README.md

```diff
@@ -59,6 +61,38 @@ NOTE: `VULKAN_LOADER_GENERATED_DIR` should be the directory that contains `vk_la
 
 See [docker/build.Dockerfile](docker/build.Dockerfile) for detailed Ubuntu build instructions.
 
+## Enabling the layers:
+To enable the performance layers, set these environent variables:
+
+`$LD_LIBRARY_PATH`: Need to append the directory containing the .so files of performance layers to the `$LD_LIBRARY_PATH`.
+```
+export LD_LIBRARY_PATH=<path-with-layers.so>:$LD_LIBRARY_PATH
+```
+`$VK_INSTANCE_LAYERS`: Specify which layers from the following set to enable.
+1. VK_LAYER_STADIA_pipeline_compile_time
+2. VK_LAYER_STADIA_pipeline_runtime
```

Comment:
nit: we can repeat `1.` for each item and markdown will automatically generate correct indices. This way we don't have to worry we get them wrong if we add more items to the list in the future.